### DISCUSSION TITLE
Openssl 1.1.1t, 3.0.9 and 3.1.1 are out

### DISCUSSION THREAD

**Discussion by rootnk** [quictls/openssl#116] [General]
how quickly are you working on the +quictls versions?

**Comment by gene-git** [comment]
I tried to build 3.0.9+quic branch bit it fails with a compile error (incorrect number of arguments) however the 3.1.0 branch compiles and seems to be working okay in preliminary testing.   I don't as yet see 3.1.1+quictls but will test once its available.

**Reply by rootnk** [reply]
Thank you

**Comment by tmshort** [comment]
We did release a 3.0.9 and 1.1.1u (and 1.1.1t, earlier).
However, we have not looked at 3.1.x yet.
3.2 is a completely different beast; and may require a different approach.

Semantic versioning is weird regarding the naming of the tags, which is why `+`s were used, but that causes other issues, so we moved to `-` and that violates semantic versioning naming requirements.
